src/app/components/wrapper-components/tutorial-game-wrapper/tutorial-game-wrapper.wrapper.component.spec.ts

```diff
@@ -975,6 +982,24 @@ describe('TutorialGameWrapperComponent (wrapper)', () => {
             const sixTutorial: TutorialStep[] = new SixTutorial().tutorial;
             const yinshTutorial: TutorialStep[] = new YinshTutorial().tutorial;
             const stepExpectations: [Rules<Move, AbstractGameState>, TutorialStep, Move, MGPValidation][] = [
+                [
+                    new ApagosRules(ApagosState),
+                    apagosTutorial[2],
+                    ApagosMove.drop(ApagosCoord.ZERO, Player.ZERO),
+                    MGPValidation.failure($localize`This move is a drop, please do a transfer!`),
+                ],
+                [
+                    new ApagosRules(ApagosState),
+                    apagosTutorial[3],
+                    ApagosMove.drop(ApagosCoord.TWO, Player.ZERO),
+                    MGPValidation.failure($localize`You actively made your opponent win!`),
+                ],
+                [
+                    new ApagosRules(ApagosState),
+                    apagosTutorial[3],
+                    ApagosMove.transfer(ApagosCoord.THREE, ApagosCoord.TWO).get(),
+                    MGPValidation.failure($localize`Wrong choice, your opponent will win next turn whatever piece is dropped!`),
```

Comment:
will win *in* the next turn?
et whatever piece -> no matter which piece

Reply:
Fait!